src/incidents/view/ViewIncident.tsx

```diff
@@ -1,31 +1,142 @@
 
 import useAddBreadcrumbs from '../../page-header/breadcrumbs/useAddBreadcrumbs'
 import { useUpdateTitle } from '../../page-header/title/TitleContext'
 import useTranslator from '../../shared/hooks/useTranslator'
+import Note from '../../shared/model/Note'
+import Permissions from '../../shared/model/Permissions'
+import NewNoteModal from '../../shared/notes/NewNoteModal'
 import { RootState } from '../../shared/store'
+import { uuid } from '../../shared/util/uuid'
+import useAddIncidentNote from '../hooks/useAddIncidentNote'
+import useDeleteIncidentNote from '../hooks/useDeleteIncidentNote'
+import useIncident from '../hooks/useIncident'
+import useResolveIncident from '../hooks/useResolveIncident'
+import NotesTable from './NotesTable'
 import ViewIncidentDetails from './ViewIncidentDetails'
 
 const ViewIncident = () => {
-  const { id } = useParams()
-  const { permissions } = useSelector((root: RootState) => root.user)
+  const { id } = useParams() as any
+  const location = useLocation()
+  const history = useHistory()
   const { t } = useTranslator()
+  const { permissions, user } = useSelector((root: RootState) => root.user)
+  const { data, isLoading } = useIncident(id)
+  const [mutate] = useResolveIncident()
+  const [mutateAddNote] = useAddIncidentNote()
+  const [deleteNote] = useDeleteIncidentNote()
+
   const updateTitle = useUpdateTitle()
-  updateTitle(t('incidents.reports.view'))
+  const [showNewNoteModal, setShowNoteModal] = useState<boolean>(false)
+  const newNoteState = {
+    id: uuid(),
+    givenBy: user?.id,
+    text: '',
+    date: '',
+  }
+  const [editedNote, setEditedNote] = useState<Note>(newNoteState)
   useAddBreadcrumbs([
     {
       i18nKey: 'incidents.reports.view',
       location: `/incidents/${id}`,
     },
   ])
 
+  updateTitle(t('incidents.reports.view'))
+
+  const onNewNoteClick = () => {
+    setEditedNote(newNoteState)
+    setShowNoteModal(true)
+  }
+  const closeNewNoteModal = () => {
+    setShowNoteModal(false)
+  }
+
+  const onResolve = async () => {
+    await mutate(data)
+    history.push('/incidents')
+  }
+
   if (id === undefined || permissions === undefined) {
     return <></>
   }
 
-  return <ViewIncidentDetails incidentId={id} permissions={permissions} />
+  return (
+    <div>
+      <ViewIncidentDetails isLoading={isLoading} incident={data} />
+      <TabsHeader>
+        <Tab
+          active={location.pathname === `/incidents/${id}/notes`}
+          label={t('patient.notes.label')}
+          onClick={() => history.push(`/incidents/${id}/notes`)}
+        />
+      </TabsHeader>
+      <Panel>
+        <div className="col-md-12 d-flex mb-3 justify-content-end">
+          <Button
+            outlined
+            color="success"
+            icon="add"
+            iconLocation="right"
+            onClick={onNewNoteClick}
+            className="create-new-note-button"
+          >
+            {t('patient.notes.new')}
+          </Button>
```

Comment:
Related to the comment I made on permissions during testing, but we should check the user's permissions to ensure that they're allowed to report incidents.